static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx

```diff
@@ -0,0 +1,138 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import OnboardingIntegrationSection from 'sentry/components/events/featureFlags/onboardingIntegrationSection';
+import {AuthTokenGeneratorProvider} from 'sentry/components/onboarding/gettingStartedDoc/authTokenGenerator';
+import type {OnboardingLayoutProps} from 'sentry/components/onboarding/gettingStartedDoc/onboardingLayout';
+import {Step, StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import type {DocsParams} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {useSourcePackageRegistries} from 'sentry/components/onboarding/gettingStartedDoc/useSourcePackageRegistries';
+import {useUrlPlatformOptions} from 'sentry/components/onboarding/platformOptionsControl';
+import ConfigStore from 'sentry/stores/configStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {space} from 'sentry/styles/space';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export function FeatureFlagOnboardingLayout({
+  docsConfig,
+  dsn,
+  platformKey,
+  projectId,
+  projectSlug,
+  newOrg,
+  projectKeyId,
+  configType = 'onboarding',
+  integration = '',
+  provider = '',
+}: OnboardingLayoutProps) {
```

Comment:
Does it even make sense to re-use `OnboardingLayoutProps`?
There are a few props included that are never used or have static values in this use case.
see `newOrg`, `activeProductSelection`, `configType`

Reply:
i think it's easier/cleaner to reuse `OnboardingLayoutProps`, but i'll create an extension type as you suggested instead of modifying this type